Tasks/Gradle/CodeAnalysis/Common/BaseTool.ts

```diff
@@ -46,26 +46,12 @@ export abstract class BaseTool implements IAnalysisTool {
      */
     protected abstract parseXmlReport(xmlReport: string, moduleName: string): [number, number];
 
-    public configureBuild(toolRunner: ToolRunner): ToolRunner {
-        switch (this.buildOutput.buildEngine) {
-            case BuildEngine.Maven: {
-                tl.debug("Not implemented");
-                throw new Error();
-            }
-            case BuildEngine.Gradle: {
-
-                if (this.isEnabled()) {
-                    console.log(tl.loc('codeAnalysis_ToolIsEnabled'), this.toolName);
-
-                    var initScriptPath: string = path.join(__dirname, '../', this.toolName.toLowerCase() + '.gradle');
-                    toolRunner.arg(['-I', initScriptPath]);
-                    break;
-                }
-            }
-        }
-        return toolRunner;
-    }
-
+    /**
+     * Configures the provided ToolRunner instance with arguments which will invoke the tool represented by this class.
+     * @param toolRunner
+     * @returns {ToolRunner} ToolRunner instance with arguments applied
+     */
+    public abstract configureBuild(toolRunner: ToolRunner): ToolRunner;
```

Comment:
Made this abstract, as the generalizable Gradle approach does not apply to Maven.